packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -142,6 +141,48 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
 
+    final codeWidget = OutlinedBorder(
+      child: Column(
+        children: [
+          debuggerSectionTitle(theme,
+              text: script == null ? ' ' : '${script.uri}'),
+          Expanded(
+            child: CodeView(
+              script: script,
+              stack: controller.currentStack.value,
+              controller: controller,
+              lineNumberToBreakpoint: _breakpointsForLines(),
+              onSelected: toggleBreakpoint,
+            ),
+          ),
+        ],
+      ),
+    );
+
+    final codeArea = ValueListenableBuilder(
+      valueListenable: controller.librariesVisible,
+      builder: (context, value, _) {
```

Comment:
rename value to visible to make it more clear what value is in the context of this block

Reply:
will do